tests/test_core_whitelist.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+import pytest
+from urlchecker.core.whitelist import white_listed
+
+
+@pytest.mark.parametrize('test_url', ["https://github.com/SuperKogito/spafe/issues/1"
+                                      "https://github.com/SuperKogito/spafe/issues/2",
```

Comment:
If these are the repos, we would want to use the official testing repository (or a subfolder of it) here.

Reply:
These are just test urls but I agree using urls related to the project is better. Which tells me that maybe we need to change the urls in the [urlchecker-test-repo](https://github.com/urlstechie/urlchecker-test-repo).

Reply:
Yes! That's the idea - since we have the subfolder variable, you can create different testing scenarios based on subfolders there. E.g.,:

```
# urlchecker-test-repo
   subfolder1
   subfolder2
   ...
   subfolderN
```
Of course you would give them meaningful names, and then for the test you can set them as variables.